js/src/forum/utils/getCleanDisplayName.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Whether to use the old mentions format.
+ *
+ * `'@username'` or `'@"Display name"'`
+ */
+export const ShouldUseOldFormat = () => app.forum.data.attributes.allowUsernameMentionFormat || false;
+
+const GetDeletedUserText = () => app.translator.trans('core.lib.username.deleted_text');
```

Comment:
shouldn't non components not be capitalised ? might just be a react thing, but It's a good rule I believe.

Reply:
Yeah we use camel case for everything, this should continue to use it.